src/tools/kata-ctl/src/check.rs

```diff
@@ -100,7 +102,38 @@ pub fn check_cpu_attribs(
     Ok(missing_attribs)
 }
 
-pub fn run_network_checks() -> Result<()> {
+#[tokio::main]
+pub async fn run_network_checks() -> Result<()> {
+    println!("Running network checks...");
+    let (connection, handle, _) = rtnetlink::new_connection().context("failed to create netlink connection").unwrap();
+    let thread_handler = tokio::spawn(connection);
+    defer!({
+        thread_handler.abort();
+    });
+
+    let host_unique_id = String::from(&kata_sys_util::rand::UUID::new());
```

Comment:
Also, do not write code like `aaa::bbb::ccc::ddd()`, write `use aaa::bbb:::ccc:::ddd` and `ddd()` instead.